packages/core/stdlib/datetime/main.browse

```diff
@@ -1,14 +1,29 @@
+#*  @name { DateTime }
+# @scope { DateTime utilities }
 import math
 
 import "./native.js"
 
-set SECOND 1000
-set MINUTE $SECOND * 60
-set HOUR $MINUTE * 60
-set DAY $HOUR * 24
-set WEEK $DAY * 7
+#*Number of seconds in a minute
```

Comment:
Space after `#*` please

Also, I'd love it if you (in a new PR) can add this to the `@browselang/format` package so we can start formatting BrowseDoc :)

Reply:
Create an issue and self-assign if you're doing something else and don't want to do this right now. The formatter is fun to work with though :)